ta_services/keymaster/km.c

```diff
@@ -5,6 +5,27 @@
 
 #include "km.h"
 
+bool version_info_set = false;
+uint32_t boot_os_version = 0;
+uint32_t boot_os_patchlevel = 0;
+
+TEE_Result km_configure(uint32_t os_version, uint32_t os_patchlevel)
+{
+	IMSG("os_version = %u", os_version);
+	IMSG("os_patchlevel = %u", os_patchlevel);
+
+	if (!version_info_set) {
+		//https://android.googlesource.com/trusty/app/keymaster/+/994293cc45700fa58512b312c94da0f46d95403e
+		// Note that version info is now set by Configure, rather than by the bootloader.  This is
+		// to ensure that system-only updates can be done, to avoid breaking Project Treble.
+		boot_os_version = os_version;
```

Comment:
This depends on using `TA_FLAG_INSTANCE_KEEP_ALIVE` (missing) to work as intended

Reply:
Thanks! Will add to `user_ta_header_defines.h`.